Gopkg.toml

```diff
@@ -22,7 +22,8 @@
 
 [[constraint]]
   name = "github.com/cloudtrust/common-service"
-  version = "v1.2.1"
+  branch = "jira-2064"
```

Comment:
why this PR ? 
Once we have a release, this will not be needed, no?

Reply:
You can't build new version of keycloak-bridge is the bridge uses common-service/jira-2064 and bridge uses keycloak-client which is using common-service/v1.2.1
Of course, we'll have to either merge `branch="master"` or `version="v1.x.y"` but not this commit with `jira-2064`